webui/tests/selenium/webui-selenium-test.py

```diff
@@ -410,8 +392,20 @@ def close_alert_and_get_its_text(self, accept=True):
         return alert_text
 
     def tearDown(self):
-        self.driver.quit()
-        self.assertEqual([], self.verificationErrors)
+        if(os.environ.get('TRAVIS') == 'true'):
+            print("Link to job : https://saucelabs.com/jobs/%s" % self.driver.session_id)
+            sauce_client = SauceClient(self.sauce_username, self.access_key)
+            try:
+                if sys.exc_info() == (None, None, None):
+                    sauce_client.jobs.update_job(self.driver.session_id, passed=True)
+                else:
+                    sauce_client.jobs.update_job(self.driver.session_id, passed=False)
+            finally:
+                self.driver.quit()
+                self.assertEqual([], self.verificationErrors)
+        else:
+            self.driver.quit()
+            self.assertEqual([], self.verificationErrors)
```

Comment:
`sys.exc_info()` is pythons method to check if exceptions were thrown. If there were none we pass the test success to saucelabs by using the sauceclient module.